diff --git a/dist/runtime/server/middleware/i18n-loader.js b/dist/runtime/server/middleware/i18n-loader.js
index 6b7b6be5720d791cb766313f37928dccb77fbe13..ece0dc094e54a11691cd43de01c1e19d4ca943a9 100644
--- a/dist/runtime/server/middleware/i18n-loader.js
+++ b/dist/runtime/server/middleware/i18n-loader.js
@@ -45,8 +45,11 @@ export default defineEventHandler(async (event) => {
   });
   let translations = {};
   for (const translationPath of paths) {
+    console.log('translationPath', translationPath)
     try {
       const fileContent = await readFile(translationPath, "utf-8");
+      console.log('fileContent', fileContent)
+
       const content = JSON.parse(fileContent);
       if (isEmptyObject(translations)) {
         translations = content;
diff --git a/dist/runtime/translationHelper.js b/dist/runtime/translationHelper.js
index 850045ec8ed59d1f3036f66ee7fbd32ef22e5433..1542045ba1078641459b00bea5bf0b9343a08806 100644
--- a/dist/runtime/translationHelper.js
+++ b/dist/runtime/translationHelper.js
@@ -58,7 +58,9 @@ export function useTranslationHelper() {
       };
     },
     hasGeneralTranslation(locale) {
-      return !!generalLocaleCache[locale];
+      const generalCache = generalLocaleCache[locale];
+      console.log('hasGeneralTranslation generalCache', generalCache)
+      return !!generalCache
     },
     hasPageTranslation(locale, routeName) {
       return !!routeLocaleCache[`${locale}:${routeName}`];
