diff --git a/dist/module.mjs b/dist/module.mjs
index 590f109fcaef71954a18cd69da8e4be1898bf596..c27b7e54a5ac651713b1b0804b8a85d2d75aa786 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -437,7 +437,7 @@ const module = defineNuxtModule({
         throw new Error("Nuxt-18n-micro: Some locale codes does not match customRegexMatcher");
       }
     }
-    nuxt.options.runtimeConfig.i18nConfig = {
+    nuxt.options.runtimeConfig.i18nConfig ??= {
       rootDir: nuxt.options.rootDir,
       rootDirs
     };
